sdk/textanalytics/Azure.AI.TextAnalytics/src/SingleLabelClassifyOptions.cs

```diff
@@ -35,8 +35,13 @@ public SingleLabelClassifyOptions()
         /// </para>
         /// </summary>
         /// <remarks>
-        /// This property only applies for <see cref="TextAnalyticsClientOptions.ServiceVersion.V2022_05_01"/> and up.
+        /// This property only applies for <see cref="TextAnalyticsClientOptions.ServiceVersion.V2022_05_01"/> and newer.
```

Comment:
was this meant to be added to DisplayName?

Reply:
No, it's on the right property, but like you're comment above, since SingleLabelClassify wasn't even added until 2022-05-01 this is redundant and probably confusing.

Reply:
I've removed this remark from single-, multi-label classify, and recognize custom entities since those were added in 2022-05-01 anyway. The comment is redundant.